test/unit/spec/common/cellComponents/tabs/jobStateListRow-Spec.js

```diff
@@ -1,78 +1,180 @@
 define([
-    'base/js/namespace',
     'common/cellComponents/tabs/jobStatus/jobStateListRow',
-    'jquery',
     'common/props',
+    '../../jobsData',
     'json!../../../../../data/testAppObj.json',
-], (Jupyter, jobStateListRow, $, Props, TestAppObject) => {
+], (jobStateListRow, Props, JobsData, TestAppObject) => {
     'use strict';
 
-    describe('The job status tab module', () => {
+    const model = Props.make({
```

Comment:
This is the test file that was driving me round the twist. Originally I had sets of tests to ensure that `updateState` and `updateParams` function correctly and reject inappropriate data, but things were failing inexplicably so I have removed them for now. Ideally I would like to put them back in as the issue is to do with Jasmine/Karma, which is not a good reason to omit tests!

Reply:
The failing tests have been reinstated as I found out where the problems were.